### ISSUE TITLE
Example using abcrown to verify visual transformer (ViT)

### ISSUE THREAD

**Issue by chiraagChakravarthy** [Verified-Intelligence/alpha-beta-CROWN#65]
I've seen that abcrown now has the ability to verify self attention, and transformers. I am wondering if you could provide an example for verifying transformer architecture using this library.
Thank you for your continued support for this library.

**Comment by Lohit106** [comment]
I am also facing the same issue - getting unsupported operations for ViT model
Please could you provide an example for verifying transformer architecture using this library.

**Comment by shizhouxing** [comment]
Here is a config for verifying ViT: https://github.com/Verified-Intelligence/alpha-beta-CROWN/blob/main/complete_verifier/exp_configs/vnncomp23/vit.yaml.

Benchmark repo for ViT: https://github.com/shizhouxing/ViT_vnncomp2023 
The benchmark is used in [VNN-COMP 2023](https://github.com/ChristopherBrix/vnncomp2023_benchmarks) and [VNN-COMP 2024](https://github.com/ChristopherBrix/vnncomp2023_benchmarks).

Paper: https://arxiv.org/abs/2405.21063

**Comment by Lohit106** [comment]
Thank you for your email
Here is the issue I am getting..
Unsupported operations error

ERROR    11:24:32     Unsupported operations:
ERROR    11:24:32     Name: onnx::LayerNormalization, Attr: {'axis': -1,
'epsilon': 1e-06}
ERROR    11:24:32     Name: onnx::LayerNormalization, Attr: {'axis': -1,
'epsilon': 1e-06}
ERROR    11:24:32     Name: onnx::Erf, Attr: {}
ERROR    11:24:32     Name: onnx::LayerNormalization, Attr: {'axis': -1,
'epsilon': 1e-06}

Here is the model with its hyperparameters I used

model = create_model('vit_base_patch16_224', pretrained=True, num_classes=2)

criterion = nn.CrossEntropyLoss()

optimizer = optim.Adam(model.parameters(), lr=0.001)

**Comment by shizhouxing** [comment]
@Lohit106 
You may refer to the ViT model in our benchmark which does not have the unsupported operators:
https://github.com/shizhouxing/ViT_vnncomp2023/blob/master/vit.py
